tests/e2e/main_test.go

```diff
@@ -12,7 +12,7 @@ import (
 	"testing"
 
 	_ "github.com/go-sql-driver/mysql"
-	_ "github.com/lib/pq"
+	_ "github.com/jackc/pgx/v4/stdlib"
```

Comment:
Can you remove both pgx and mysql here?

And then add the that mysql import to this file?

```
internal/testdb/mariadb.go
``` 

The e2e tests are calling into `testdb` (this is where the drivers are required), so these blank imports are no longer needed here. Likely left over from a previous PR.

Reply:
No problem! Done. Thanks!